lib/deprecated-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@storybook/api",
+  "version": "6.5.0-alpha.64",
+  "description": "Event names used in storybook core",
```

Comment:
```suggestion
  "description": "Deprecated manager API",
```

Reply:
Maybe we should put this in `lib/deprecated/api`?